src/Microsoft.Framework.DesignTimeHost/DesignTimeAssemblyLoadContextFactory.cs

```diff
@@ -62,9 +57,12 @@ private IAssemblyLoadContextFactory GetRuntimeFactory(IServiceProvider servicePr
 
                 // Add a cache dependency on restore complete to reevaluate dependencies
                 ctx.Monitor(_namedDependencyProvider.GetNamedDependency(_project.Name + "_Dependencies"));
-
-                return new AssemblyLoadContextFactory(applicationHostContext.ServiceProvider);
+                
```

Comment:
@troydai OK. Then do this in VS Code (if you have not don this already): 
1) Ctrl + P type Settings
2) Select - "Preferences: Open User Settings"
3) Paste this:

// Place your settings in this file to overwrite the default settings
{
	"editor.renderWhitespace": true,
	"editor.insertSpaces": "auto"
}

The result: You will see white spaces (tabs vs. spaces) as in VS. If you do tab it will insert spaces if the file uses spaces (hence auto).